Makefile

```diff
@@ -49,8 +49,5 @@ elvis:
 cover:
 	@rebar3 cover -v
 
-coveralls:
-	@rebar3 coveralls send
```

Comment:
It would be nice to do something with code coverage though. Perhaps we can put it on GitHub pages or something... (not in scope for this PR)

Reply:
Yes, I used coveralls in my own fork only since it required you to give it some rights to access the repo, something that wasnt possible to do in Nordix at that time. Now there is at least a printout in log, but with coveralls you could have a banner showing the coverage in the README.md that was nice